package.json

```diff
@@ -51,9 +65,12 @@
   "devDependencies": {
     "assert": "^1.4.1",
     "delay": "^4.1.0",
-    "eslint": "^5.9.0",
-    "eslint-plugin-mocha": "^6.0.0",
+    "eslint": "^7.4.0",
```

Comment:
we can't use our npm packages here right? :)

Reply:
Correct.  What's done here is essentially the same thing, just local.  In the future, we will probably just open source our linting rules so they can be used everywhere.